opencti-platform/opencti-front/src/private/components/settings/users/User.js

```diff
@@ -513,8 +513,10 @@ class UserComponent extends Component {
               />
             </Paper>
           </Grid>
+          <Grid item={true} xs={6} style={{ paddingTop: 8 }}>
```

Comment:
Proposition: Isn't it more readable with more space ?

paddingTop: 40 px on the lest grid

![image](https://user-images.githubusercontent.com/53513584/230024594-86223054-7efa-4dec-ab4c-d0265c4592b7.png)

Reply:
you're right! maybe not that much for paddingTop, but I will increase it to 25 ou 30 :)